model/src/main/java/com/basistech/rosette/dm/Dependency.java

```diff
@@ -32,7 +32,7 @@
  * </pre>
  * Each token has two columns related to dependencies: the 'governor' and the 'label'. In the ADM,
  * this would turn into a list of {@code Dependency} objects as follows:
- * <table>
+ * <table summary="An example list of Dependencies">
```

Comment:
This is to satisfy java 8's javadoc requirements (as revealed by running `mvn javadoc:javadoc`). Apparently this is an accessibility feature of HTML.